src/components/structures/LoggedInView.js

```diff
@@ -71,6 +72,10 @@ export default React.createClass({
         // RoomView.getScrollState()
         this._scrollStateMap = {};
 
+        // Only run these in electron, at least until a better mechanism for perms exists
+        // https://w3c.github.io/permissions/#dom-permissionname-device-info
+        if (window && window.process && window.process.type) CallMediaHandler.loadDevices();
```

Comment:
PR is to fix #2822 primarily so that's what its designed for, I did no testing in a browser but should work given microphone+camera permissions at browser level